db/migrate/20231118125941_activate_voucher_by_default.rb

```diff
@@ -0,0 +1,5 @@
+class ActivateVoucherByDefault < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
The change method is also executed when rolling back a migration. But in that case we don't want the feature to be enabled again. Let's change it to `up`.
```suggestion
  def up
```